packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -69,15 +72,24 @@ contract("ServerlessHub.js", function (accounts) {
     return request(`http://localhost:${port}`).post("/").send(body).set("Accept", "application/json");
   };
 
+  const startGanacheServer = (chainId, port) => {
+    const node = ganache.server({ _chainIdRpc: chainId });
+    node.listen(port);
+    return new Web3("http://127.0.0.1:" + port);
+  };
+
   before(async function () {
+    accounts = await web3.eth.getAccounts();
+    [contractDeployer] = accounts;
     defaultChainId = await web3.eth.getChainId();
+    await runDefaultFixture(hre);
```

Comment:
ideally, in the future, we should move away from these kinds of fixtures as they ended up slowing things down more than need be by deploying more than required a lot of the time.

Reply:
Yep! There are a bunch of simplicity-vs-speed tradeoffs here. I basically just use this when any tests (intentionally or not) depend on pre-deployed contracts. I don't include it by default, though, so most of our tests actually don't need it, which means we get most of this speedup already!